### DISCUSSION TITLE
MRbase webapp down?

### DISCUSSION THREAD

**Discussion by SoroushMJ** [MRCIEU/TwoSampleMR#578] [Q&A]
![webapp](https://github.com/user-attachments/assets/821fdc1b-ac30-4bd5-9839-34fe41aa22cd)

Hi all,

It seems the web app has been down for a few days (please see the attached error for reference). Is anyone else experiencing this issue, or is there an update planned that I might have missed?

Thanks in advance for your help!

**Comment by liyao-jzmu** [comment]
The same problem has been going on for a long time
